about.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- <script
+      src="https://kit.fontawesome.com/a076d05399.js"
+      crossorigin="anonymous"
+    ></script> -->
+
+    <link rel="stylesheet" href="./styles.css" />
+
+    <title>English Learning Course</title>
+  </head>
+  <body>
+    <!-- Navigation section -->
+    <div class="main-top-header">
```

Comment:
- [ ] Please make sure It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.